ansible/roles/snapshot_snapper/tasks/main.yml

```diff
@@ -36,17 +40,18 @@
 
 # Configure updatedb to exclude snapshots [[[
 - name: 'Divert original configuration under /etc'
-  command: dpkg-divert --quiet --local --divert "{{ item }}.dpkg-divert" --rename "{{ item }}"
-  args:
-    creates: '{{ item }}.dpkg-divert'
+  dpkg_divert:
+    path: '{{ item }}'
   register: snapshot_snapper__updatedb_diverted_register
   when: ansible_os_family in [ "Debian" ]
   loop: '{{ snapshot_snapper__divert_files | flatten }}'
 
 - name: Copy diverted configuration file to original location
-  command: cp {{ item }}.dpkg-divert {{ item }}
-  args:  # noqa no-handler
```

Comment:
Since this task shouldn't be done as a handler (well, we could add it as a handler and do meta: flush_handlers but that's not really needed here...), make sure to copy the `# noqa no-handler` comment to tell `ansible-lint` to ignore this issue.